utilities/bin/core/commands.dsc

```diff
@@ -0,0 +1,503 @@
+                                    - adjust <player> skin_blob:<player.flag[jlib.puppet.skin]>
+                                    - flag player jlib.puppet:!
+                                - else:
+                                    - flag player jlib.puppet:<map.with[name].as[<player.name>].with[skin].as[<player.skin_blob>].with[npc].as[<[npc]>]>
+                                    - teleport <player> <[npc].location>
+                                    - adjust <player> display_name:<[npc].nickname>
+                                    - adjust <player> health:<[npc].health>
+                                    - adjust <player> skin_blob:<[npc].skin_blob>
+                                    - adjust <player> gamemode:survival
+                                    - despawn <[npc]>
+                            - else:
+                                - narrate "<[name].equals[closest].if_true[<[colors].get[error]>Unable to find an npc within 5 blocks of you!].if_false[<[colors].get[soft]><[name]> <[colors].get[error]>is not a valid npc name!]>"
+                        - else:
+                            - narrate <proc[lib_core_command_error].context[missing_keys|puppet|<context.args.proc[lib_core_command_extra_keys]>]>
+                    - else:
+                        - narrate <proc[lib_core_command_error].context[extra_keys|puppet|<[arg_map].keys.proc[lib_core_command_extra_keys]>]>
+                - else:
+                    - narrate <proc[lib_core_command_error].context[max_args|puppet|1|<context.args.size>]>
+            - else:
+                - narrate <proc[lib_core_command_error].context[implicit]>
+        - else:
+            - narrate <proc[lib_core_command_error].context[permission]>
+
+#Ends all currently running queues
+end_queues:
+    type: command
+    name: end_queues
+    debug: false
+    description: Allows you to end all currently running queues on the server.<&nl>WARNING<&co> This is a nuclear option! This is only meant to be used in cases where you have infintely looping queues that you want to end without wanting to restart the whole server. Do not use this command within a script!
+    usage: <proc[lib_command_usage].context[lib_generic_data|command.usage.end_queues]>
+    aliases:
+        - killqueues
+        - kill_queues
+        - killqs
+    allowed help:
+        - determine <proc[lib_has_permission].context[utilities.end_queues|<player.if_null[null]>].if_null[true]>
+    script:
+        - if <proc[lib_has_permission].context[utilities.end_queues|<player.if_null[server_or_command_block]>].if_null[true]>:
+            - if <player.exists>:
+                - if <context.args.size> == 0:
+                    - define warn <script[lib_config].parsed_key[color.warning]>
+                    - define soft <script[lib_config].parsed_key[color.soft_server_notice]>
+                    - if <queue.list.size> > 1:
+                        - foreach <queue.list.exclude[<queue>]> as:queue:
```

Comment:
These two tags are deprecated in favor of [<util.queues>](https://meta.denizenscript.com/Docs/Search/util.queue)
```suggestion
                    - if <util.queue.size> > 1:
                        - foreach <util.queues.exclude[<queue>]> as:queue:
```

Reply:
Yes, this is also true.